docs/install.md

```diff
@@ -103,7 +103,7 @@ Unlocking the bootloader on modern Samsung devices have some caveats. The newly
 
 ### Instructions
 
-- Use either [samfirm.js](https://github.com/jesec/samfirm.js), [Frija](https://forum.xda-developers.com/s10-plus/how-to/tool-frija-samsung-firmware-downloader-t3910594), or [Samloader](https://forum.xda-developers.com/s10-plus/how-to/tool-samloader-samfirm-frija-replacement-t4105929) to download the latest firmware zip of your device directly from Samsung servers.
+- Use either [samfirm.js](https://github.com/jesec/samfirm.js), [Frija](https://forum.xda-developers.com/s10-plus/how-to/tool-frija-samsung-firmware-downloader-t3910594),[Samloader](https://forum.xda-developers.com/s10-plus/how-to/tool-samloader-samfirm-frija-replacement-t4105929) or [Samloader kotlin](https://github.com/zacharee/SamloaderKotlin) to download the latest firmware zip of your device directly from Samsung servers.
```

Comment:
Not a native English speaker, but AFAIK, there should be a space after a comma.

```suggestion
- Use either [samfirm.js](https://github.com/jesec/samfirm.js), [Frija](https://forum.xda-developers.com/s10-plus/how-to/tool-frija-samsung-firmware-downloader-t3910594), [Samloader](https://forum.xda-developers.com/s10-plus/how-to/tool-samloader-samfirm-frija-replacement-t4105929), or [Samloader kotlin](https://github.com/zacharee/SamloaderKotlin) to download the latest firmware zip of your device directly from Samsung servers.
```

Reply:
Yeah, I made a pull request from my phone. That's why I forgot to put the comma. lol . 

I think you should remove samfirm.js and Samloader from the documentation because these projects are no longer maintained.